packages/ra-ui-materialui/src/input/useSupportCreateSuggestion.tsx

```diff
@@ -19,24 +19,28 @@ import set from 'lodash/set';
  * @param options The hook option
  * @param {ReactElement} options.create A react element which will be rendered when users choose to create a new choice. This component must call the `useCreateSuggestionContext` hook which provides `onCancel`, `onCreate` and `filter`. See the examples.
  * @param {String} options.createLabel Optional. The label for the choice item allowing users to create a new choice. Can be a translation key. Defaults to `ra.action.create`.
- * @param {String} options.createItemLabel Optional. The label for the choice item allowing users to create a new choice when they already entered a filter. Can be a translation key. The translation will receive an `item` parameter. Defaults to `ra.action.create_item`.
+ * @param {String} options.createItemLabel Optional. The label for the choice item allowing users to create a new choice when they already entered a filter. Can be a translation key. The translation will receive an `item` parameter. Providing this option will turn the create label when there is no filter to be a hint (i.e. a disabled item).
  * @param {any} options.createValue Optional. The value for the choice item allowing users to create a new choice. Defaults to `@@ra-create`.
+ * @param {any} options.createHintValue Optional. The value for the (disabled) item hinting users on how to create a new choice. Defaults to `@@ra-create-hint`.
  * @param {String} options.filter Optional. The filter users may have already entered. Useful for autocomplete inputs for example.
  * @param {OnCreateHandler} options.onCreate Optional. A function which will be called when users choose to create a new choice, if the `create` option wasn't provided.
  * @param {Function} options.handleChange A function to pass to the input. Receives the same parameter as the original event handler and an additional newItem parameter if a new item was create.
  *
  * @returns {UseSupportCreateValue} An object with the following properties:
  * - getCreateItem: a function which will return the label of the choice for create a new choice.
+ * - handleChange: a function which should be called when the input value changes. It will call the `onCreate` function if the value is the createValue.
  * - createElement: a React element to render after the input. It will be rendered when users choose to create a new choice. It renders null otherwise.
+ * - getOptionDisabled: a function which should be passed to the input to disable the create choice when the filter is empty (to make it a hint).
  */
 export const useSupportCreateSuggestion = (
     options: SupportCreateSuggestionOptions
 ): UseSupportCreateValue => {
     const {
         create,
         createLabel = 'ra.action.create',
-        createItemLabel = 'ra.action.create_item',
+        createItemLabel,
```

Comment:
One could argue this is technically a BC, but I thought it was worth it to reuse the `createItemLabel` prop to decide whether or not to make the `createLabel` option clickable, rather than introducing yet another prop.
Besides, this hook can probably be considered an internal hook, as it is not documented.